src/Compilers/CSharp/Portable/Binder/Binder.ValueChecks.cs

```diff
@@ -22,6 +22,87 @@ private enum EscapeLevel : uint
             ReturnOnly = ReturnOnlyScope,
         }
 
+        /// <summary>
+        /// Encapsulates a property or indexer and the accessor(s) on it that were used. The particular
```

Comment:
It looks like this is used for regular methods as well as accessors. Consider updating the comment to reflect that.